### PR TITLE
docs: improve Azure DevOps setup documentation

### LINKED ISSUE #8332: syntax/code highlighting broken on Azure DevOps docs page
**What Renovate type, platform and version are you using?**

Problem is with documentation at https://docs.renovatebot.com/setup-azure-devops/

**Describe the bug**

The code blocks are not rendering properly.
This problem was raised by **viceice** as a reviewer for PR https://github.com/renovatebot/renovate/pull/8269#issuecomment-761598609.

**Relevant debug logs**

**Have you created a minimal reproduction repository?**

Please read the [minimal reproductions documentation](https://github.com/renovatebot/renovate/blob/master/docs/development/minimal-reproductions.md) to learn how to make a good minimal reproduction repository.

- [x] This is a really small bug, it does not need a reproduction (think small typo)
- [ ] I have provided a minimal reproduction repository
- [ ] I don't have time for that, but it happens in a public repository I have linked to
- [ ] I don't have time for that, and cannot share my private repository
- [ ] The nature of this bug means it's impossible to reproduce publicly

**Additional context**

I'm making a PR to fix this problem.

### PR DESCRIPTION
**Pull request by HonkingGoose**
## Changes:

It's basically a full rewrite.

- Fix syntax highlighting problem by using a simpler style of ordering the document
- Add page headings for clearer navigation
- Add introductory sentence explaining purpose of document
- Move sentences around
- Improve explanation for which parts of the files you need to change
- Fix some typos
- Remove some fluff words

## Context:

The problem of bad syntax highlighting was raised by viceice as a reviewer https://github.com/renovatebot/renovate/pull/8269#issuecomment-761598609, and they asked for my help to fix it.

Closes #8332 

## Documentation (please check one with an [x])

- [x] I have updated the documentation, or
- [ ] No documentation update is required

## How I've tested my work (please tick one)

I have verified these changes via:

- [x] Code inspection only, or
- [ ] Newly added unit tests, or
- [ ] Unit tests + ran on a real repository

### CODE REVIEW

File: docs/usage/setup-azure-devops.md [hunk 1]
```diff
@@ -5,72 +5,91 @@ description: How to setup Renovate for Azure DevOps
...
+      npx --userconfig .npmrc renovate
+    env:
+      TOKEN: $(System.AccessToken)
+```
+
+## Create a .npmrc file
+
+Create a `.npmrc` file in your repository:
+
+```json
```
Comment (viceice) [inline docs/usage/setup-azure-devops.md:51]:
```suggestion
```ini
```

It's an ini file 😏

Comment (HonkingGoose) [reply docs/usage/setup-azure-devops.md]: Thank you! 😄

File: docs/usage/setup-azure-devops.md [hunk 2]
```diff
@@ -5,72 +5,91 @@ description: How to setup Renovate for Azure DevOps
...
-
-If you wish to use a single Renovate pipeline to update multiple repositories:
-
-1. Add the names of the repositories to `config.js`.
-
-1. Make sure that the "Project Collection Build Service (YOUR-PROJECT)" user has the following permissions on the repositores:
-   - Contribute
-   - Contribute to pull requests
-   - Create branch
+This document explains how to setup Renovate for use on Azure DevOps.
```
Comment (HonkingGoose) [inline docs/usage/setup-azure-devops.md:8]:
```suggestion
This document explains how to setup Renovate for use with Azure DevOps.
```

I'm not sure if it should be for "use _on_" or "for use _with_".

File: docs/usage/setup-azure-devops.md [hunk 3]
```diff
@@ -5,72 +5,91 @@ description: How to setup Renovate for Azure DevOps
...
+## Create a .npmrc file
+
+Create a `.npmrc` file in your repository:
+
+```ini
+registry=https://pkgs.dev.azure.com/YOUR-ORG/_packaging/YOUR-FEED/npm/registry/
+always-auth=true
+```
+
+In the `registry` variable, replace `YOUR-ORG` with your Azure DevOps organization and `YOUR-FEED` with your Azure Artifacts feed.
```
Comment (HonkingGoose) [inline docs/usage/setup-azure-devops.md:56]:
```suggestion
For the `registry` key, replace `YOUR-ORG` with your Azure DevOps organization and `YOUR-FEED` with your Azure Artifacts feed.
```

File: docs/usage/setup-azure-devops.md (hunk 2 above)
Comment (JamieMagee) [reply docs/usage/setup-azure-devops.md:8]: The Azure DevOps style guide says "on Azure DevOps"

**Review by JamieMagee — APPROVED** [review]

File: docs/usage/setup-azure-devops.md (hunk 2 above)
Comment (HonkingGoose) [reply docs/usage/setup-azure-devops.md:8]: Thanks you! 😄

**Pull request merged**